README.md

```diff
@@ -8,6 +8,10 @@ Notes
 ===
 * Heavily cribbed from https://github.com/bluenviron/gortsplib/blob/main/examples/client-read-format-h264-convert-to-jpeg/main.go
 
+FFmpeg
```

Comment:
Do we need to change the readme instructions under `Prep linux` to say that the apt packages must only be installed on the build machine but that the prod machine doesn't need to have those packages?

Reply:
Good call! We no longer need apt installs. The build system expects local FFmpeg build. Edited this section with build instructions and a note on static build.